includes/misc-functions.php

```diff
@@ -23,9 +23,20 @@
  */
 function give_is_test_mode() {
```

Comment:
@mehul0810 As discussed over call please improve this function and check multiple scenarios like:
1. Donation form page
2. Donation form shortcode on page
3. Multiple donation form shortcode on page